Sources/Operators/ConcatMap.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  ConcatMap.swift
+//  CombineExt
+//
+//  Created by Daniel Peter on 22/11/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+#if canImport(Combine)
+import Combine
+import class Foundation.NSRecursiveLock
```

Comment:
no reason for this, Combine already uses Foundation. I think import Foundation would be fine here.